CHANGES.md

```diff
@@ -16,6 +16,12 @@
 
 [//]: # (list changes here, using '-' for each new entry, remove this when items are added)
 
+[1.17.5](https://github.com/bird-house/birdhouse-deploy/tree/1.17.5) (2021-11-16)
```

Comment:
You should just added the changes stuff but do not touch this line.  The `bump2version` command will do it for you.

Remove the line above `[//]: # (list changes here, using '-' for each new entry, remove this when items are added)` as well, `bump2version` will restore it for the next change.